internal/services/storage/storage_share_resource_test.go

```diff
@@ -264,6 +552,19 @@ func TestAccStorageShare_protocolUpdate(t *testing.T) {
 }
 
 func (r StorageShareResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
```

Comment:
Can we use the `!features.FivePointOhBeta` flag here to control the correct behaviour